collectd/build.gradle

```diff
@@ -24,7 +24,14 @@ plugins {
 }
 
 apply from: "$buildScriptsDir/common-java.gradle"
-apply from: "$buildScriptsDir/publishing.gradle"
+
+jar {
+    classifier = 'standalone'
```

Comment:
As we're only distributing CollectD as a self-contained jar I'm not sure we need this classifier here.
If we decide to use it anyway then I think it would be better to investigation whether there is standard naming convention for such distributions